pkg/storaged/pages.jsx

```diff
@@ -608,8 +608,8 @@ export const PageTable = ({ emptyCaption, aria_label, pages, crossrefs, sorted,
                 </Card>);
         } else {
             const cols = [
-                <Td key="1" onClick={onClick}>
-                    <div className={"content-level-" + level}>
+                <Td key="1" onClick={onClick} modifier="nowrap">
+                    <div style={{ paddingInlineStart: level + "rem" }}>
```

Comment:
I think we want to keep the `content-level-` however. That's also probably more performant then inline styling @garrett ?

Reply:
> I think we want to keep the content-level- however.

Why?

Reply:
Let's keep it with the content-level for now. I would propose we just "nest" > 10 under the 10th indent so it doesn't look too weird?

Reply:
What? It's _horrible_ to have inline CSS like this.

Why not just use CSS custom properties (aka: variable)?

Here's what the CSS would look like:

```css
padding-inline-start: calc(var(--content-level) * var(--pf-v5-global--spacer--md));
```

Here's how you set it via HTML:

```html
<div class="indent" style="--content-level: 2">Hi</div>
```

And if you're chaning it via JavaScript, here's an example (of course, change the selector):

```js
document.querySelector('.indent').style.setProperty('--content-level', 5);
```

Reply:
If the variable is completely unset (that is, not set anywhere else that would apply via the cascade), the CSS would not resolve, so it would not apply at all. You could also have a fallback like `padding-inline-start: calc(var(--content-level, 0) * var(--pf-v5-global--spacer--md));` — that is, on the var like this: `var(--value, FALLBACK)`

Reply:
> It's _horrible_ to have inline CSS like this.

Why?

Reply:
> Why not just use CSS custom properties (aka: variable)?

Thanks for the suggestion! I have implemented it like so:
```
.indent {
  padding-inline-start: calc(var(--pf-v5-global--spacer--md) * min(var(--level, 0), 10));
  white-space: nowrap;
}
```
This caps the indentation at 10. Looks like this:

![image](https://github.com/cockpit-project/cockpit/assets/3228183/eb6da476-1a70-46fb-a090-9603baa8397a)

Reply:
Ah, using `min()` like that is a clever idea, and it's nice to have it "just work" without adding additional code to check it... and keeping it in the style/layout area (the CSS, in other words :wink:) is also the right place for that! `white-space: nowrap;` might cause issues for longer subvolume names... we probably would want it to wrap, otherwise there might be a horizontal scroll.

Reply:
Aha, it gets truncated:

![image](https://github.com/cockpit-project/cockpit/assets/10246/bbe9667a-ba09-4c18-9660-b076dcdd5082)

I guess there's no way to know which is which though, unless I were to click on them and go to another page.